app/src/main/java/app/revanced/integrations/youtube/patches/VideoInformation.java

```diff
@@ -178,8 +197,22 @@ public static boolean seekTo(final long seekTime) {
             }
 
             Logger.printDebug(() -> "Seeking to " + adjustedSeekTime);
+
+            // Try calling the seekTo method of the MDX player director first (called when casting to a big screen device).
+            // The difference has to be at least 1000ms to avoid infinite skip loops (the seekTo Lounge API command only supports seconds).
+            if (adjustedSeekTime - videoTime >= 1000) {
+                try {
+                    mdxSeekMethod.invoke(mdxPlayerDirectorRef.get(), adjustedSeekTime);
```

Comment:
It does return a boolean indeed, and the method has the same signature as the other one, so I assume that it would also behave in a similar fashion - but since I haven't bothered to check the return values, I only injected this extra function call and swallowed the exceptions. Will let you know if I end up doing testing on this.

Reply:
Done.